### PR TITLE
Added event enums, endpoints to check if this account has any trusted…

### PR DESCRIPTION
**Pull request by rayandrie**
… device and endpoint to remove current device as trusted, customization text and images

### CODE REVIEW

File: Example/CotterIOS/EntryViewController.swift [hunk 1]
```diff
@@ -121,6 +128,23 @@ class EntryViewController: UIViewController {
         })
     }
     
+    @IBAction func enrollTrustedDeviceForUser(_ sender: Any) {
```
Comment (albertpurnama) [inline Example/CotterIOS/EntryViewController.swift:131]: Why do you need this here?

**Review by albertpurnama — APPROVED** [review]
Tried on iPhone X, works well.

**Comment by albertpurnama** [comment]
Don't forget to write out the docs!

File: Example/CotterIOS/EntryViewController.swift (hunk 1 above)
Comment (rayandrie) [reply Example/CotterIOS/EntryViewController.swift:131]: cuz my phone was already a registered user but i haven't enrolled in trusted devices yet so i couldn't login. Can prob take it out if u dont like

**Pull request merged**